docs/installation.md

```diff
@@ -0,0 +1,100 @@
+# Installation Guide
+
+There are a few steps to run CodeX Editor on your site.
+
+## Load Editor's core
+
+> Installation via NPM/Yarn will be added soon.
+
+Copy [codex-editor.js](../build/codex-editor.js) file to your project and load it.
+
+```html
+<script src="codex-editor.js"></script>
+```
+
+## Load external Tools
+
+Load Tools with their classes and styles.
```

Comment:
Each Block at the CodeX Editor represented by Tools. There are simple (or powerful) external scripts with own logic. To start using the Editor, you should connect minimum number of Tools.